openslides/users/rest_api.py

```diff
@@ -0,0 +1,13 @@
+from openslides.utils.rest_api import RESTElement
+
+from . import views
+
+
+class UserRESTElement(RESTElement):
+    viewset = views.UserViewSet
+
+
+class GroupRESTElement(RESTElement):
+    viewset = views.GroupViewSet
+    app_label = 'users'
+    router_prefix = 'group'
```

Comment:
The queryset of the GroupViewSet contains the model. The name of the model `group`. It is just like all the other cases you dealed with (Motions, Categories, Users, Items, ...).